src/main/java/com/sendgrid/helpers/mail/Mail.java

```diff
@@ -266,6 +268,16 @@ public void addContent(Content content) {
     this.content = addToList(newContent, this.content);
   }
 
+  /**
+   * Take the html part of content and convert it to plain text. Add it to content
+   * @param html part of content
+     */
+  public void addHtmlAsText(String html) {
+    Document doc = Jsoup.parse(html);
+    String text = doc.text();
+    addContent(new Content("text/plain", text));
+  }
```

Comment:
This looks good, but I'd like to see a test that validates this behavior works as expected.